src/main/java/com/example/bulletinboardapp/repository/UsersRepo.java

```diff
@@ -0,0 +1,10 @@
+package com.example.bulletinboardapp.repository;
+
+import com.example.bulletinboardapp.entity.User;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+import java.util.Optional;
+
+public interface UsersRepo extends JpaRepository<User,Integer> {
```

Comment:
```suggestion
public interface UserRepo extends JpaRepository<User,Integer> {
```
в единственном числе